test/common/router/config_impl_test.cc

```diff
@@ -53,7 +53,9 @@ class TestConfigImpl : public ConfigImpl {
                  bool validate_clusters_default)
       : ConfigImpl(config, factory_context, validate_clusters_default), config_(config) {}
 
-  RouteConstSharedPtr route(const Http::HeaderMap& headers, uint64_t random_value) const override {
+  RouteConstSharedPtr route(const Http::HeaderMap& headers,
```

Comment:
I'm probably missing it, but could you point me to where the tests for the new behaviors are?

Reply:
I've just now pushed the unit-test for tls-context matching.